frontend/src/Components/Elements/Post.tsx

```diff
@@ -26,12 +28,23 @@ function Post({
 }: PostProps) {
   return (
     <div className="timeline-box flex flex-col overflow-clip">
+      {/* Add conditional rendering for UsernameRepost */}
+      <div className="-mx-3 mb-4 flex flex-row justify-end border-b border-gray-500 px-6 pb-1">
```

Comment:
```suggestion
      <div className="-mx-3 mb-4 flex flex-row justify-end border-b border-black25 px-6 pb-1 dark:border-white25">
```
We've used the black25/50/75 and white25/50/75 colors in most places for consistency. Probably good to use them here too.